package.json

```diff
@@ -143,10 +143,6 @@
     "test:integration": "node --expose-gc --max-old-space-size=4096 --experimental-vm-modules --trace-deprecation node_modules/jest-cli/bin/jest --logHeapUsage --testMatch \"<rootDir>/test/*.{basictest,longtest,test}.js\"",
     "test:basic": "node --expose-gc --max-old-space-size=4096 --experimental-vm-modules --trace-deprecation node_modules/jest-cli/bin/jest --logHeapUsage --testMatch \"<rootDir>/test/*.basictest.js\"",
     "test:unit": "node --max-old-space-size=4096 --experimental-vm-modules --trace-deprecation node_modules/jest-cli/bin/jest --testMatch \"<rootDir>/test/*.unittest.js\"",
-    "travis:integration": "yarn cover:integration --ci $JEST",
-    "travis:basic": "yarn cover:basic --ci $JEST",
-    "travis:lintunit": "yarn lint && yarn cover:unit --ci $JEST",
-    "travis:benchmark": "yarn benchmark --ci",
     "appveyor:integration": "yarn cover:integration --ci %JEST%",
```

Comment:
These also look unused now. I could add a second commit to clean them up or a separate PR

Reply:
I think better to do it here

Reply:
OK, added the extra commit to remove them too